plugins/example-todo-list-backend/package.json

```diff
@@ -17,12 +17,11 @@
   "publishConfig": {
     "access": "public",
     "main": "dist/index.cjs.js",
-    "types": "dist/index.d.ts",
-    "alphaTypes": "dist/index.alpha.d.ts"
+    "types": "dist/index.d.ts"
```

Comment:
This one keeps main but removes alphaTypes - the inverse of the other one 🤔

Reply:
There wasn't actually any alpha exports in this one, so no need to move over to using `exports`